opennms-base-assembly/src/main/filtered/etc/org.opennms.plugin.elasticsearch.rest.forwarder.cfg

```diff
@@ -0,0 +1,54 @@
+# org.opennms.plugin.elasticsearch.rest.forwarder.cfg
```

Comment:
Since this is just documentation, I would omit adding this to the base assembly.